ui/components/multichain/notification-detail-nft/index.scss

```diff
@@ -0,0 +1,4 @@
+.notification-detail-nft__image {
+  width: 112px;
+  height: 112px;
+}
```

Comment:
You may be able to use `width` and `height` attributes instead of these ; the height/width are based 4, so 28.